examples/annotate-config/README.md

```diff
@@ -0,0 +1,22 @@
+# Annotate Config
+
+The `annotate-config` KRM config function adds an annotation to configuration.
+This example invokes the annotate-config function using declarative
+configuration.
+
+## Function invocation
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/annotate-config .
+kpt fn run annotate-config/configs --fn-path annotate-config/functions
```

Comment:
Why do you need `--fn-path`. Try to show the simplest CLI usage.

Reply:
As with set-namespace, it's cleaner not to add an annotation to the function config. I only want add an annotation to the configs in the package dir so I put the function config in a separate fn-path.

Reply:
This is trying to bypass a deeper issue here: You don't want the function to manipulate the function config.

Reply:
I agree it's a bypass but let's merge these examples now and update them later after there's consensus around a solution like the no-process annotation.